lib/views/auth_page.dart

```diff
@@ -1,12 +1,126 @@
+                            borderRadius: BorderRadius.circular(10)),
+                      ),
+                    ),
+                    const SizedBox(
+                      height: 37,
+                    ),
+                    TextField(
+                      obscureText: true,
+                      decoration: InputDecoration(
+                        fillColor: Colors.grey.shade50,
+                        filled: true,
+                        hintText: 'Enter your password',
+                        border: OutlineInputBorder(
+                            borderRadius: BorderRadius.circular(10)),
+                      ),
+                    ),
+                    const SizedBox(
+                      height: 40,
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        const Text(
+                          'Login',
+                          style: TextStyle(
+                            fontSize: 30,
+                            fontFamily: 'Moon',
+                          ),
+                        ),
+                        CircleAvatar(
+                          radius: 30,
+                          backgroundColor: const Color(0x00000000),
+                          child: IconButton(
+                            color: Colors.white,
+                            onPressed: () => Get.to(const MainPage()),
+                            icon: const Icon(Icons.arrow_forward_ios_outlined),
+                          ),
+                        )
+                      ],
+                    ),
+                    const SizedBox(
+                      height: 20,
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        TextButton(
+                          onPressed: () {},
+                          child: const Text(
+                            'Sign Up',
+                            textAlign: TextAlign.left,
+                            style: TextStyle(
+                                decoration: TextDecoration.underline,
+                                fontFamily: 'Moon',
+                                color: Color(0xff4c505b),
+                                fontSize: 18),
+                          ),
+                          style: const ButtonStyle(),
+                        ),
+                        TextButton(
+                            onPressed: () {},
+                            child: const Text(
+                              'Forgot Password?',
+                              style: TextStyle(
+                                decoration: TextDecoration.underline,
+                                fontFamily: 'Moon',
+                                fontSize: 18,
+                                color: Color(0xff4c505b),
```

Comment:
Keep all the defined colors inside a file (eg: color_constants.dart) inside widgets folder and then import and use it.

/For code Modularity